use_raphrrr_sfc/module_surface.f90

```diff
@@ -450,18 +450,18 @@ subroutine remove_snow(this,rapfile,rrfsfile,rlat)
              if( (ix >= 1 .and. ix <= nx_rap) .and. &
                  (jx >= 1 .and. jx <= ny_rap) ) then
                 if(tmp2d4b(ix,jx) < 1.0e-10_r_single .and. rlat(i,j) < 37.0 ) then
-                   tmp2d8b(i,j)=0.0_r_kind
+                   tmp2d4br(i,j)=0.0_r_kind
```

Comment:
Why to add an extra "r" here? Will "tmp2d4b" without "r" be more consistent with others?